### DISCUSSION TITLE
SCPA Q-value validations and FC relations

### DISCUSSION THREAD

**Discussion by MAGBP** [jackbibby1/SCPA#79] [General]
Hi Jack
I've been loving SCPA and the extra power and resolution it brings to single cell pathway analysis. I'm a young student using it for my master thesis on methotrexate pharmacodynamics in psoriatic arthritis, and it has helped me find some really interesting stuff. During this process though, I've been wondering about a few things:

1. Do you have any in vitro validation studies I could use for Q-value interpretation, and for reference? 

2. What do you think about interpreting FC values independently of Q-value? I know these two have a natural relation in some fashion, but what would you say about a pathway that has a Q-value of 1.8 but a FC of 5 for example? Is there a 'hard stop' for Q-values where you discard a pathway and it's FC?

3. Also, do you have experience running the same dataset on different gene sets, and how do you feel this generally affects Q-values?

Kind regards and thank you in advance!

**Comment by jack-bibby** [comment]
Hi,

Sorry, I must have missed this. It may be a bit belated now but...

1. See Figure 3 and 4 in [our paper](https://www.cell.com/cell-reports/fulltext/S2211-1247(22)01571-6?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124722015716%3Fshowall%3Dtrue) where there are a few in vitro experiments validating the Qval
2. I would generally avoid directly using the fold change values themselves and just use them as an indicator of any potential enrichment. I would focus on using the Qval as this will be the meaningful measure of differential regulation in a pathway RE [this explanation](https://jackbibby1.github.io/SCPA/articles/interpreting_scpa_output.html#which-output-values-should-you-use)
3. I'm not sure I fully follow this question -- any new pathway you test will generate a new Qval that is independent of another pathway

Jack
